docs/get-started/developers/installation.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: layout-with-subnav.njk
+title: Installation
+heading: Developers
+tags:
+  - developers
+permalink: /get-started/developers/installation/index.html
+order: 10
+bodyClasses: element-docs
+---
+
+## How to install
+
+To develop components or design system documentation, you must first install some required software, namely node.js. We use [nvm](https://github.com/nvm-sh/nvm) to ensure a uniform development environment.
+
+### Step 1: Install node
+
+**Fedora/RHEL users** should install nvm for bash directly from GitHub.
+
+<rh-code-block>
+  <script type="text/sample-javascript">
```

Comment:
Please check `<script>` tags throughout for these code blocks. I'm not sure I added the right MIME types...